### PR TITLE
Default Grid Assesment

### PR DESCRIPTION
**Pull request by Ali-Tehrani**
This pull-requests addresses Issue #179.
The Default rgrid was obtained from [Horton](https://github.com/theochem/horton/blob/master/data/grids/tv-13.7-5.txt). This corresponds to using the UniformInteger grid and PowerRTransform with a "fine" preset for the angular component. This is stored in grid/utils.py in a dictionary with the following:
```python
r"""
Obtained from theochem/horton/data/grids/tv-13.5.txt with the folllowing preamble

These grids were created by Toon Verstraelen in July 2013 in a second effort
to generate tuned grids based on a series of diatomic benchmarks computed
with the QZVP basis of Ahlrichs and coworkers. They are constructed to give
on average 5 digits of precision for a variety of molecular integrals, using
the Becke scheme with k=3.
"""
# Items are (rmin, rmax, npts) in angstrom, Meant for PowerRTransform
```

If the user provided rgrid as None, then it generates the default radial grid.
The following functions and their tests (with two added) were changed to have default rgrid options:
- AtomGrid.from_preset
- MolGrid.from_preset
- MolGrid.from_size
- MolGrid.from_pruned

### CODE REVIEW

**Comment by Ali-Tehrani** [comment]
Added default BeckeWeights to aim_weights to all static, constructor methods

File: src/grid/molgrid.py [hunk 1]
```diff
@@ -628,3 +654,33 @@ def __getitem__(self, index: int):
...
+
+    Returns
+    -------
+    OneDGrid:
+        One-dimensional grid that was transformed using PowerRTransform.
+
+    """
+    if atnum in _DEFAULT_POWER_RTRANSFORM_PARAMS:
+        rmin, rmax, npt = _DEFAULT_POWER_RTRANSFORM_PARAMS[int(atnum)]
+        rmin, rmax = rmin * 1.8897259885789, rmax * 1.8897259885789
```
Comment (FarnazH) [inline src/grid/molgrid.py:679]: 1. Please use `scipy.constants.angstrom / scipy.constants.value("atomic unit of length")` for unit conversion.
2. Please add a comment that r values are converted from Angstom to Bohr.

**Review by FarnazH — CHANGES_REQUESTED** [review]
Thanks @Ali-Tehrani.
1. Shouldn't we make the `rgrid` in `from_pruned` method optional as well?
2. Thinking more after our discussion, I agree that we should remove `*_` in arguments. It seems to have been used arbitrarily and as we are not using it anywhere else, it is best to remove it for now. Can you please remove them?

File: src/grid/molgrid.py (hunk 1 above)
Comment (Ali-Tehrani) [reply src/grid/molgrid.py]: This is fixed in the following comments

**Comment by Ali-Tehrani** [comment]
The *_ is removed.

If you're referring to AtomGrid.from_prune, default radial grid isn't added because atomic numbers is not included in its signature.

**Review by FarnazH — APPROVED** [review]
I removed `*` from the rest of AtomGrid class, which caused an API change. If you have no further comments, feel free to merge this PR.

**Comment by marco-2023** [comment]
I did not find a way to add a commit to this PR. The only change that I would  make is to substitute the values in line 198
```
                rmin, rmax = rmin * 1.8897259885789, rmax * 1.8897259885789
```
 by the corresponding scipy constants 

```
                # convert angstrom to bohr
                ang2bohr = scipy.constants.angstrom / scipy.constants.value("atomic unit of length")
                rmin, rmax = rmin * ang2bohr, rmax * ang2bohr
```

just for consistency with the molgrid module.

**Comment by FarnazH** [comment]
> I did not find a way to add a commit to this PR. The only change that I would make is to substitute the values in line 198 of AtomGrid:
> 
> ```
>                 rmin, rmax = rmin * 1.8897259885789, rmax * 1.8897259885789
> ```
> 
> by the corresponding scipy constants
> 
> ```
>                 # convert angstrom to bohr
>                 ang2bohr = scipy.constants.angstrom / scipy.constants.value("atomic unit of length")
>                 rmin, rmax = rmin * ang2bohr, rmax * ang2bohr
> ```
> 
> just for consistency with the molgrid module.

@marco-2023, Ali already did this change in [c6ff8f4](https://github.com/theochem/grid/pull/210/commits/c6ff8f43eac0ef323d4a601ec86532b78ea8e630).

**Pull request merged**